README.md

```diff
@@ -1,6 +1,6 @@
 # Portable Computing Language (PoCL)
 
-PoCL is being developed towards an efficient implementation of the OpenCL
+PoCL is a conformant implementation of the OpenCL 3.0
```

Comment:
Please add a link to the conformance report.

Reply:
Also perhaps add a different style of marker for conformant devices (CPU on X86) in the matrix?

Reply:
fixed, will be in next PR